sdk/helper/pluginidentityutil/fields_test.go

```diff
@@ -115,25 +115,34 @@ func TestPopulatePluginIdentityTokenData(t *testing.T) {
 	}
 }
 
-func TestAddPluginIdentityTokenFields(t *testing.T) {
+func TestAddPluginIdentityTokenFieldsWithGroup(t *testing.T) {
 	testcases := []struct {
 		name  string
+		group string
 		input map[string]*framework.FieldSchema
 		want  map[string]*framework.FieldSchema
 	}{
 		{
 			name:  "basic",
 			input: map[string]*framework.FieldSchema{},
+			group: "default",
```

Comment:
Do we want to use a group name other than default just to make sure we're using it?